examples/nrunner/recipes/runnable/identifier.json

```diff
@@ -0,0 +1 @@
+{"kind": "noop", "identifier": "1-test-noop"}
```

Comment:
I think having the example identifier here starting with "1-" makes it easy to confuse that with a [Test ID](https://avocado-framework.readthedocs.io/en/latest/guides/user/chapters/concepts.html#test-id)'s first component.